packages/flutter_tools/lib/src/web/compile.dart

```diff
@@ -48,9 +49,22 @@ Future<void> buildWeb(FlutterProject flutterProject, String target, BuildInfo bu
         flutterProject.manifest.appName,
         '${fs.path.withoutExtension(target)}_web_entrypoint.dart.js',
       );
+      // Check for deferred import outputs.
+      final File dart2jsArchive = fs.file(fs.path.join(
+          flutterProject.dartTool.path,
+          'build',
+          'flutter_web',
+          '${flutterProject.manifest.appName}',
+          '${fs.path.withoutExtension(target)}_web_entrypoint.dart.js.tar.gz'));
```

Comment:
nit: trailing comma for better readability?

Reply:
Fixed